plugin/src/test/java/org/jenkinsci/plugins/workflow/SerializationTest.java

```diff
@@ -101,43 +99,6 @@ public boolean start() throws Exception {
         }
     }
 
-    /**
-     * Workflow captures a stateful object, and we verify that it survives the restart
-     */
-    @Test public void persistEphemeralObject() throws Exception {
-        story.addStep(new Statement() {
-            @Override public void evaluate() throws Throwable {
-                jenkins().setNumExecutors(0);
-                DumbSlave s = createSlave(story.j);
-                String nodeName = s.getNodeName();
-
-                p = jenkins().createProject(WorkflowJob.class, "demo");
-                p.setDefinition(new CpsFlowDefinition(
-                    "def s = jenkins.model.Jenkins.instance.getComputer('" + nodeName + "')\n" +
-                    "def r = s.node.rootPath\n" +
```

Comment:
Seems to have been relying on pickles deprecated since https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/180.